### ISSUE TITLE
build with Travis

### ISSUE THREAD

**Issue by plata** [PhoenicisOrg/WineBuild#3]
Couldn't we just use Travis to build the Wine versions we need? It's really simple to specify Linux and OSX builds in the `.yml` so why not use this instead of Docker?

**Comment by plata** [comment]
@alesliehughes are you the new point of contact for Wine staging? If so, I'd be interested in your plans regarding Wine and Wine staging builds. Maybe we could join forces to achieve builds which can also be used in Phoenicis PlayOnLinux/PlayOnMac?

**Comment by alesliehughes** [comment]
Yes, I am the new point of contact.  My plans would to continue to release just like wine every fortnight.  I'm still working on getting the ability to created packages. I don't have any plans to work with the PlayOnLinux/Mac. They have created binaries for staging in the past and would expect that to start again once a release occurs.

**Comment by plata** [comment]
Are you somehow testing the commits/pull requests for Wine staging or do you just build locally?

It's true that we built (and still build) the binaries for Wine and Wine staging. However, the build system is somewhat messed up and we are looking for a better solution. The new solution should be capable of building all releases of Wine and Wine staging for Linux and OSX. Therefore I was thinking that you might have similar requirements to test your releases.

As you see in this issue, I'm considering Travis as one possibility. Another possibility could be a Docker based build service.

**Comment by alesliehughes** [comment]
We build locally and test locally. 
We have a builder at https://dev.wine-staging.com, which can build/publishes packages for us.

**Comment by plata** [comment]
Do you think it would be feasible to build Wine (staging) in Travis? I'm thinking about something like an automated process to:
- fetch patch
- apply patch
- create pull request
- build

As for the builder service: Can it build only .deb for Debian or would it be possible to get .zip? Also, we would need a machine readable file which contains the available version like this:  http://phoenicis.playonlinux.com/index.php/wine?os=linux

**Comment by plata** [comment]
I'm trying to build Wine with Travis here: https://travis-ci.org/plata/wine

The build is failing because of Vulkan (even though it should be disabled). Any ideas?

**Comment by ImperatorS79** [comment]
Install vulkan dev files ?

BTW why are you using gcc 4.9.2 instead of 7.2 ?

(You're also missing libhal and OSS but I think you know it :-))

**Comment by plata** [comment]
I know but Travis uses Ubuntu Trusty and that doesn't seem to provide vulkan etc. Maybe I must use docker to get a newer Ubuntu version.

**Comment by plata** [comment]
Now in: https://github.com/plata/wine-builds
